### PR TITLE
added fjord-foundry

### PR DESCRIPTION
**Pull request by thehashmap**

### CODE REVIEW

File: src/questions/fjord-foundry.yaml [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+- uuid: 2f1ea1ea-2bc3-403d-8caa-33155fc9403e
+  type: SingleChoice
+  content: What is the goal of setting a high initial price in a Liquidity Bootstrapping Pool (LBP)?
+  hint: NoHint
+  explanation: Setting a high initial price allows the changing pool weights of your LBP to fully effect the price, gradually dropping it until market equilibrium is established. Unlike previous token sale methods, such as bonding curves, consumers are disincentivized to buy early and instead gain from waiting for the price to fall until it hits a level they consider fair.
+  answerKeys:
+    - A
+  difficultyLevel: Low
+  choices:
+    - content: To sell the token for more than it is worth
```
Comment (RobinNagpal) [inline src/questions/fjord-foundry.yaml:61]: Not sure if this is a right choice.

"Consumers are disincentivized to buy early, so it prevents majority of tokens getting sold to whales, thus helping in uniform distribution of the tokens and creating and effective community"

File: src/questions/fjord-foundry.yaml [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+  explanation: Redemption pools are special pools that allow projects to progressively distribute a token over a defined time period in exchange for a specified base token (In some cases this could be releasing a governance token in exchange for LP). During the Redemption Pool period, users can exchange the base token for the project's token, which contributes to the collective decision of the asset's fair value.
+  answerKeys:
+    - C
+  difficultyLevel: Low
+  choices:
+    - content: Users pay for a governance token in exchange for liquidation tokens.
+      key: A
+    - content: Users can exchange the project's token for the base token.
+      key: B
+    - content: Users can exchange the base token for the project's token.
```
Comment (RobinNagpal) [inline src/questions/fjord-foundry.yaml:150]: Include "progressively distribute a token over a defined time period" as that is very important point

File: src/questions/fjord-foundry.yaml [hunk 3]
```diff
@@ -0,0 +1,170 @@
+- uuid: fed2ef38-e906-4e1b-93bf-d4ec5bdf3ac3
```
Comment (RobinNagpal) [inline src/questions/fjord-foundry.yaml:1]: Add a questions

- Which of these is not a configuration option while creating a new LBP?

File: src/questions/fjord-foundry.yaml (hunk 1 above)
Comment (thehashmap) [reply src/questions/fjord-foundry.yaml:61]: Oh that's a mistake, I marked the wrong option as answer. Correct answer should be option B.

**Pull request merged**